GitCommands/Git/GitModule.cs

```diff
@@ -2460,9 +2460,9 @@ public IReadOnlyList<GitItemStatus> GetStashDiffFiles(string stashName)
             // shows untracked files
             var args = new GitArgumentBuilder("log")
             {
-                $"{stashName}^3",
                 "--pretty=format:\"%T\"",
-                "--max-count=1"
+                "--max-count=1",
+                $"-- {stashName}^3"
```

Comment:
https://github.com/gitextensions/gitextensions/pull/9056#discussion_r616185182
https://git-scm.com/docs/git-log#_description

stashName is a revision, not a path and should not be after "--"
The "fatal:" outputs need to be handled

Reply:
I noticed your argument. But I guess git might have a different definition of _paths_ because it works now without error.
Do you have repro where this changes causes wrong behavior?

Reply:
No repro, only Git documentation. 
All other commands with "--" are supposed to be followed by paths, so also if this works, it could be changed.
If I am voted down, this must at least be clearly commented.

Reply:
You are absolute right. I got the git error wrong. I have replaced this with a check for `ExitedSuccessfully`.
Else stashed untracked files were not displayed anymore.
![grafik](https://user-images.githubusercontent.com/36601201/118717815-d06cee80-b826-11eb-8d7d-f374e7c37e82.png)